data_collections/csv_updater.py

```
@@ -1,4 +1,5 @@
 from .events import getEvents
+from .internships import getInternships
```

Ruff (0.11.9)

`.internships.getInternships` imported but unused

Remove unused import: `.internships.getInternships`

(F401)